app/assets/javascripts/discourse/app/lib/sidebar/admin-nav-map.js

```diff
@@ -110,6 +110,12 @@ export const ADMIN_NAV_MAP = [
         label: "admin.community.sidebar_link.legal",
         icon: "gavel",
       },
+      {
+        name: "admin_moderation_flags",
```

Comment:
I do wonder if for now we should hide this behind a `admin_flagging_ui_enabled_groups` feature flag? I only worry a little about admins going to this page since it's new then being a bit confused about why they would want to enable/disable flags.

Up to you, if you think it won't be a problem then leave it :+1:

Reply:
I feel quite confident here. The feature may not be the most useful but still can be used. Even UI is reflecting that if all flags are disabled it shows information:

<img width="626" alt="Screenshot 2024-05-29 at 11 31 34 AM" src="https://github.com/discourse/discourse/assets/72780/2af55d4f-ec0d-476b-b1a5-62c11b725c23">